scripts/hotspot.py

```diff
@@ -162,12 +205,15 @@ def process_vcf(infile,outfile,genomic_hs,hugo_hs,fid,aa='Amino_acids',pos='Prot
     vcfin.header.info.add("HotSpotAllele",'A','Integer',"Is the corresponding ALT allele on a hotspot (0 for false; 1 for true).")
     vcfout = pysam.VariantFile(outfile,'w',header=vcfin.header)
     csq_items = vcfin.header.info["CSQ"].description.split(' Format: ')[1].split('|')
+    flagged = []
     total_hotspots_found = 0
     for rec in vcfin.fetch():
         genomic_hotspot_found = False
         if rec.chrom in genomic_hs:
             for region in genomic_hs[rec.chrom]:
-                if rec.start == int(region[0]) and rec.stop == int(region[1]): # we're only marking it if the rec start and stop match exactly to the genomic regions
+                region_range = range(int(region[0]),int(region[1])+1) # Need to add one because the bed end position is inclusive (SNPs will have same start and end which will result in empty range)
```

Comment:
Yeah, unless I am mistaken, you need to subtract one from the position to get that bed half-open. So vcf:
`chr5	1295113`
is represented in bed as:
`chr5	1295112	1295113`

Reply:
BED format chromStart and rec.start are both 0 based. The reason your variant isn't showing up was because I misread that the end position was inclusive when it is exclusive. Removing the `+1` should be sufficient to match the region end with the rec.stop which is also exclusive. I was thrown off earlier as the input was not in proper BED format, as one would expect for genomic positions.

Reply:
Fixed